server/src/main/scala/de/frosner/broccoli/nomad/NomadConfiguration.scala

```diff
@@ -0,0 +1,21 @@
+package de.frosner.broccoli.nomad
+
+import com.typesafe.config.Config
+
+/**
+  * The Nomad configuration.
+  *
+  * @param url The URL to connect to to access nomad via HTTP.
+  */
+final case class NomadConfiguration(url: String)
```

Comment:
What will happen to the application if neither application.conf nor reference.conf contains this value?